WordPress/src/main/java/org/wordpress/android/datasets/SiteSettingsTable.java

```diff
@@ -29,6 +30,7 @@
     public static void createTable(SQLiteDatabase db) {
         if (db != null) {
             db.execSQL(SiteSettingsModel.CREATE_SETTINGS_TABLE_SQL);
+            db.execSQL(JetpackSettingsModel.CREATE_JP_SETTINGS_TABLE_SQL);
```

Comment:
Just that fact that we'd have to create and test a whole new store. There was some talk of it on Slack ~1 week ago.